### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by jlvFlores**
# In this PR, we:
- Removed fix cards in HTML
- Added dynamic cards using an array of objects using JavaScript
- Added popup window feature (modal), and styles to it
- Made modal responsive to screen size after 768px
#### Unnecessary changes I wanted to add:
- Renamed images to be more descriptive
- Edited image ALT text
- Removed horizontal overflow due to animations

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,8 +1,122 @@
...
+    <div class="card" data-card="${work.id}">
+      <img src="${work.image}" alt="${work.imageAlt}">
+      <div class="right-block">
+        <h3>${work.title}</h3>
+        <p>${work.description}</p>
+        <ul class="tags">
+        <li>${work.listArray[0]}</li>
+        <li>${work.listArray[1]}</li>
+        <li>${work.listArray[2]}</li>
+        </ul>
```
Comment (huseyinbiyik) [inline script.js:111]: Brilliant job using the  `insertAdjacentHTML`! But let's fix our tech stack tags. Currently it looks undefined. And please fix the overflowing tag as I marked below on the image.
Screenshot 2023-02-01 at 11 07 23 PM

File: style.css [hunk 2]
```diff
@@ -193,7 +193,6 @@ h1 {
 }
 
 /* Works Section */
```
Comment (huseyinbiyik) [inline style.css:195]: Good job with the design. 👌 But let's add some padding like on the Figma template. 
Screenshot 2023-02-01 at 11 06 10 PM
Screenshot 2023-02-01 at 11 06 18 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Works section created dynamically 👌
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @jlvFlores, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**